cmd/influxd/run/config_test.go

```diff
@@ -141,6 +141,8 @@ bind-address = ":2010"
 [[udp]]
 bind-address = ":4444"
 
+[[udp]]
```

Comment:
Environment variable overrides are applied by walking through the config struct to look for matching environment variable names. Since only one listener is added to the config for each section by default, multiple listeners are not possible with environment variables alone.